docs/geedocs/answer/4412455.html

```diff
@@ -142,9 +142,38 @@
       <li>In the Name field, enter <b>SFMapProject</b>, and click <b>Save</b>. The new map project name appears in the Asset Manager’s asset list. </li>
     </ol>
     <hr>
+
     <h2><a name="imagery_proj"></a>Define a Flat Imagery Project</h2>
     <p>The maps layer includes Google Maps layers and uses the Google Maps API from google.com. Map databases use either Plate Carrée (EPSG: 4326) and Mercator (EPSG: 3857) projections and normally require corresponding imagery resource types. In this lesson you add a flat imagery project from a previous tutorial lesson to a Mercator map database, which can then be projected as <em>Mercator on the Fly</em>. For more information, see <a href="../answer/6081069.html">Add flat imagery to Mercator map databases</a>.</p>
```

Comment:
Minor: you didn't change this, but it would be good to fix it if you have the opportunity.  I think "Plate Carre and Mercator projections" should be "Plate Carre or Mercator projections".